src/translations/sk.json

```diff
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "loged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
+    "logging_out": "Odhlásenie",
+    "sign_up": "Zaregistrovať sa",
+    "signing_up": "Registrácia",
+    "remember_password": "Zabudli ste heslo?",
+    "recover_password": "Obnoviť heslo",
+    "send_recovery_email": "Odoslať e-mail pre obnovenie",
+    "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+    "never_mind": "Nevadí",
+    "update_password": "Aktualizovať heslo",
+    "updated_password": "Aktualizácia heslá",
+    "complete_your_signup": "Dokončite registráciu",
+    "site_url_title": "Nastavenie vývoja",
+    "site_url_link_text": "Vymazať URL localhost",
+    "site_url_message": "Zdá sa, že používate lokálny server. Povedzte nám adresu URL svojho Netlify servera.",
+    "site_url_label": "Zadajte URL svojho servera Netlify",
+    "site_url_placeholder": "URL vášho Netlify servera",
+    "site_url_submit": "Nastaviť adresu URL",
+    "message_confirm": "Na váš e-mail bol odoslaný odkaz na potvrdenie, pokračujte kliknutím na tento odkaz.",
```

Comment:
This is just a suggestion, as I think this sentence might read a little bit better and more naturally, than trying to mirror the English sentence structure:

`Potvrďte kliknutím na odkaz v správe, ktorú sme Vám práve zaslali na váš email.`

Reply:
Shouldn't we specify what they are confirming?

`Potvrďte registráciu kliknutím na odkaz v správe, ktorú sme Vám práve zaslali na váš email.`

Reply:
Yeah, good catch! Makes sense 👍